### ISSUE TITLE
Request: Add "Line Directives"

### ISSUE THREAD

**Issue by nkev** [sipin/gorazor#31]
Great project! This makes it easy to move from ASP.NET MVC to GO.
One request I have is to add Go's undocumented "line directives" to the generated code so Go’s error reporting can reference your original source:

``` GO
//line index.ego:1
func RenderIndex(w io.Writer, widgets []*Widget) error {
//line index.ego:2
    _, _ = fmt.Fprintf(w, "\n\n<html>\n<body>\n  <h1>Widgets for Sale!</h1>\n\n  <ul>\n    ")
//line index.ego:8
    for _, widget := range widgets {
//line index.ego:9
        _, _ = fmt.Fprintf(w, "\n      <li>")
//line index.ego:9
        _, _ = fmt.Fprintf(w, "%v", widget.Name)
//line index.ego:9
        _, _ = fmt.Fprintf(w, " for $")
//line index.ego:9
        _, _ = fmt.Fprintf(w, "%v", widget.Price)
//line index.ego:9
        _, _ = fmt.Fprintf(w, "</li>\n    ")
//line index.ego:10
    }
//line index.ego:11
    _, _ = fmt.Fprintf(w, "\n  </ul>\n</body>\n</html>\n")
    return nil
}
```

The above code is from an [article](http://blog.gopheracademy.com/advent-2014/ego/) on the EGO templating engine.

**Comment by Wuvist** [comment]
@nkev Hi, thanks for the info.

"line directives" looks very helpful. Let me try to add it.

**Comment by runner-mei** [comment]
这个有开始做么？

**Comment by Wuvist** [comment]
@runner-mei 还木有。。。

**Comment by nkev** [comment]
What do the comments mean say in English?

**Comment by Wuvist** [comment]
This feature is finally implemented in [v1.2.2](https://github.com/sipin/gorazor/releases/tag/v1.2.2)
